docs/Epita/OCaml.mdx

```diff
@@ -43,36 +43,37 @@ nix-env -iA nixpkgs.opam
 </Tabs>
 
 
-## Initialisation de Opam
+## Initialization
+
+To initialize Opam, you can use the following command:
 
-Pour initialiser Opam, vous pouvez utiliser la commande suivante :
 ```bash
 opam init
 eval $(opam env)
 ```
 
-## Installation des packages supplémentaires
+## Install additional packages
 
-Plusieurs packages sont nécessaires pour travailler avec OCaml. Voici une liste de paquets à installer :
+Many packages are required to work with OCaml. Here is a list of packages to install:
 
 ```bash
 opam install dune utop merlin ocp-indent ocaml-lsp-server tuareg
 ```
 
-- utop : Un terminal interactif pour OCaml, pratique pour tester des fonctions rapidement.
-- merlin, ocp-indent, ocaml-lsp-server : Des outils pour pouvoir utiliser OCaml avec votre IDE favori (VSCode, Vim, Emacs, ...).
-- dune : Un outil pour compiler vos projets OCaml, nécessaire pour le projet d'AFIT
-- tuareg : Un mode pour Emacs pour l'édition de code OCaml
+- utop: An interactive terminal for OCaml, useful for quickly testing functions.
+- merlin, ocp-indent, ocaml-lsp-server: Tools to be able to use OCaml with your favorite IDE (VSCode, Vim, Emacs, ...).
+- dune: A tool to compile your OCaml projects, necessary for the AFIT project
+- tuareg: A mode for Emacs for editing OCaml code
```

Comment:
```suggestion
- tuareg: An Emacs mode for editing OCaml code
```

Reply:
Resolved in [24a239d](https://github.com/EpiMac/docs/pull/15/commits/24a239d6f807caf054a562a05eb88cf800e2e7a3)